tests/backup/backup_test.go

```diff
@@ -2722,6 +2722,260 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 	})
 })
 
+func GetbackupScheduleUID(backupName string, orgID string, ctx context.Context) (string, error) {
+	backupDriver := Inst().Backup
+	bkpUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+	backupSchInspectRequest := &api.BackupScheduleInspectRequest{
+		Name:  backupName,
+		Uid:   bkpUid,
+		OrgId: orgID,
+	}
+	resp, err := backupDriver.InspectBackupSchedule(ctx, backupSchInspectRequest)
+	log.FailOnError(err, "Failed for req - %v", backupSchInspectRequest)
+	log.InfoD("response here is %v", resp)
+	log.InfoD("resp.GetBackupSchedule().GetUid() - %v ", resp.GetBackupSchedule().GetUid())
```

Comment:
Remove these log statements and just add a single state with backup name and backup UID